File: Service/Implement/DanhMucATTPLoaiHoSoService.cs
namespace Service.Implement
{
    public class DanhMucATTPLoaiHoSoService : BaseService<DanhMucATTPLoaiHoSo, IDanhMucATTPLoaiHoSoRepository>
    , IDanhMucATTPLoaiHoSoService
    {
    private readonly IDanhMucATTPLoaiHoSoRepository _DanhMucATTPLoaiHoSoRepository;
    public DanhMucATTPLoaiHoSoService(IDanhMucATTPLoaiHoSoRepository DanhMucATTPLoaiHoSoRepository) : base(DanhMucATTPLoaiHoSoRepository)
    {
    _DanhMucATTPLoaiHoSoRepository = DanhMucATTPLoaiHoSoRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucATTPLoaiHoSo.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucATTPLoaiHoSo } from 'src/app/shared/DanhMucATTPLoaiHoSo.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucATTPLoaiHoSoService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucATTPLoaiHoSo";
    }
}

